src/script/vco-ie/main_warhammer/wh2_main_skv_clan_moulder/missions.lua

```diff
@@ -0,0 +1,150 @@
+local missions = {
+	[[
+ 		mission
+		{
+			victory_type vco_victory_type_skv_mdr_1_yes_yes_prepare_the_lab;
+			key wh_main_short_victory;
+			issuer CLAN_ELDERS;
+			primary_objectives_and_payload
+			{
+				objective
+				{
+					override_text mission_text_text_vco_skv_mdr_1_yes_yes_prepare_the_lab;
+					type SCRIPTED;
+					script_key vco_dummy;
+				}
+
+				objective
+				{
+					type CONSTRUCT_N_BUILDINGS_INCLUDING;
+					faction wh2_main_skv_clan_moulder;
+					total 1;
+
+					building_level wh2_main_special_hellpit_pits;
+				}
+
+				payload
+				{
+					effect_bundle
+					{
+						bundle_key vco_victory_payload_skv_mdr_1_yes_yes_prepare_the_lab;
+						turns 0;
+					}
+
+					effect_bundle
+					{
+						bundle_key vco_campaign_victory_dummy;
+						turns 0;
+					}
+
+					game_victory;
+				}
+			}
+		}
+	]],
+	[[
+ 		mission
+		{
+			victory_type vco_victory_type_skv_mdr_2_the_vermintide_comes;
+			key wh_main_short_victory;
+			issuer MUFFIN_MAN;
+			primary_objectives_and_payload
+			{
+				objective
+				{
+					override_text mission_text_text_vco_skv_mdr_2_the_vermintide_comes;
+					type SCRIPTED;
+					script_key vco_dummy;
+				}
+
+				objective

```

Comment:
Check if you can override text inside the DESTROY_FACTION objective inside of using a dummy. I think it is possible, if not we should update the wiki. Also add the confederation_valid flag.